wiki/Performance_points/ppv2/th.md

```diff
@@ -0,0 +1,8 @@
+---
+stub: true
+tags:
+  - ppv2
+---
+
+# ppv2
+**ppv2** คือระบบ [performance points](/wiki/Performance_points) ใหม่และระบบใหม่นี้จะดูว่าตัว[แมพเพลง](/wiki/Beatmap)มีความยากเท่าใดถึงงจะได้รับ [performance points](/wiki/Performance_points) จำนวนนั้นมันถูกคิดค้นโดย [Tom94](https://osu.ppy.sh/users/1857058) และ [osu!dev Discord server](/wiki/osu!dev_Discord_server) ก็ได้พัฒนามันต่อหลังจากนั้น
```

Comment:
```suggestion
**ppv2** คือระบบ [performance points](/wiki/Performance_points) ปัจจุบัน โดยระบบนี้ได้ดูว่าตัว[บีทแมพ](/wiki/Beatmap)มีความยากเท่าใดถึงจะได้รับ [performance points](/wiki/Performance_points) จำนวนนั้น โดยได้ถูกสร้างขึ้นโดย [Tom94](https://osu.ppy.sh/users/1857058) และได้รับการพัฒนาต่อโดยผู้ใช้ใน [osu!dev Discord server](/wiki/osu!dev_Discord_server) 
```
Multiple changes for readability. Feel free to discuss if you don't agree with something.